modules/stampcollection.py

```diff
@@ -78,16 +82,54 @@ def calculate_stamps(self):
             toi = self.utils.index[to_id]
             total_votes_by_user = self.utils.get_votes_by_user(from_id)
             if total_votes_by_user != 0:
-                score = (self.gamma * votes_for_user) / total_votes_by_user
+                score = votes_for_user / total_votes_by_user
                 users_matrix[toi, from_id_index] = score
-        for i in range(1, user_count):
-            users_matrix[i, i] = -1.0
-        users_matrix[0, 0] = 1.0
+        for i in range(user_count):
+            users_matrix[i, i] = 0
+
+        # self.log.debug(self.class_name, matrix=users_matrix)
 
         user_count_matrix = np.zeros(user_count)
         user_count_matrix[0] = 1.0  # God has 1 karma
```

Comment:
@robertskmiles eh, why `user_count_matrix` is called a matrix when it's a vector?

Reply:
I didn't name the variable but I'll refactor.

Reply:
Oh yeah, I didn't see the @ on that.  Rob had mentioned that he wrote the code and then someone went in and refactored all of it to be more readable without actually understanding what all the code did.  So I blame all weird variable names on whoever that person was.  Rob is innocent.